hack/preserve-bundle-labels.sh

```diff
@@ -6,15 +6,16 @@ KUEUE_OPERATOR_IMAGE="registry.redhat.io/kueue-tech-preview/kueue-rhel9-operator
 DESIRED_BASE="scratch"
 CSV_FILE="bundle/manifests/kueue-operator.clusterserviceversion.yaml"
 DOCKERFILE="bundle.Dockerfile"
+DISPLAY_NAME="Red Hat Build of Kueue"
 
 # Use the correct base image for the bundle.
 sed -i "s|^FROM .*|FROM ${DESIRED_BASE}|" "${DOCKERFILE}"
 
 # Insert custom labels after metrics.project_layout label.
 sed -i "/^LABEL operators.operatorframework.io.metrics.project_layout=go\.kubebuilder\.io\/v4/a \\
 \\
-LABEL io.k8s.display-name=\"OpenShift Kueue Bundle\"\\
-LABEL io.k8s.description=\"This is a bundle for the kueue operator\"\\
+LABEL io.k8s.display-name=\"Red Hat Build of Kueue operator bundle\"\\
+LABEL io.k8s.description=\"This is a bundle for the Red Hat Build of Kueue operator\"\\
```

Comment:
Just FYI, "Operator" should generally be capitalized I think, at least it is for docs

Reply:
done.